assets/javascripts/initializers/discourse-math.js.es6

```diff
@@ -3,17 +3,28 @@ import loadScript from 'discourse/lib/load-script';
 
 let initializedMathJax = false;
 
+// The site settings are grabbed in the exported initialize function.
+// I'm not sure what the best way to make that variable available to the
+// other functions in the file. I decided to introduce a variable that's
+// local to this file but available accross the file. I'm curious if there's
+// a better approach?
+// Mark McClure, while adding a zoom on click option.
+let file_site_settings;
```

Comment:
in general the way you do this is like so:

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/pretty-text/engines/discourse-markdown/code.js.es6#L31

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/pretty-text/engines/discourse-markdown/code.js.es6#L10

This allows you to avoid the global which is a risky liability.